src/components/search/components/cta-card.tsx

```diff
@@ -0,0 +1,82 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import {get} from 'lodash'
+
+import {bpMinMD} from 'utils/breakpoints'
+import {track} from 'utils/analytics'
+import ExternalTrackedLink from 'components/external-tracked-link'
+
+const CtaCard: React.FC<{location: string; trackTitle: string; resource: any}> =
```

Comment:
created a CTA Card in `/search/components` as this is a component that we are using consistently when updating instructor pages